bridge/src/log_stream.rs

```diff
@@ -119,53 +126,31 @@ impl<T: Transport> Stream for LogStream<T> {
     fn poll(&mut self) -> Poll<Option<Self::Item>, Self::Error> {
         loop {
             let (next_state, value_to_yield) = match self.state {
-                State::AwaitInterval => {
-                    // wait until `interval` has passed
-                    let _ = try_stream!(self.poll_interval.poll().chain_err(|| format!(
-                        "LogStream (topic: #{:?}): polling interval failed",
-                        self.topic
-                    )));
-                    info!(
-                        "LogStream (topic: #{:?}): polling last block number",
-                        self.topic
+                State::AwaitBlockNumber => {
+                    let last_block = try_stream!(
+                        self.block_number_stream
+                            .poll()
+                            .chain_err(|| "LogStream: fetching of last confirmed block number failed")
```

Comment:
Might be better to actually re-try the call instead of failing here, but I guess it's beyond scope of this PR, as I suspect it's probably the assumption across the entire codebase that the RPC is always available.

Reply:
yes, it is

Reply:
I believe we could fix this on a transport layer for all requests

Reply:
Yup, `ReTryingTransport` could be a thing (same as `TimeoutTransport` actually).